Google.PowerShell/BigQuery/BqCmdlet.cs

```diff
@@ -11,11 +13,17 @@ namespace Google.PowerShell.BigQuery
     /// </summary>
     public class BqCmdlet : GCloudCmdlet
     {
-        public BigqueryService Service { get; private set; }
+        private readonly Lazy<BigqueryService> _service;
+        private readonly Lazy<BigQueryClient> _client;
+
+        public BigqueryService Service => _service.Value;
+        public BigQueryClient Client => _client.Value;
 
         public BqCmdlet()
         {
-            Service = new BigqueryService(GetBaseClientServiceInitializer());
+            //Service = new BigqueryService(GetBaseClientServiceInitializer());
+            _service = new Lazy<BigqueryService>(() => new BigqueryService(GetBaseClientServiceInitializer()));
```

Comment:
Not sure if you need a Lazy for the service since it's almost always used by the cmdlet.